flows/inspect.go

```diff
@@ -9,6 +9,82 @@ import (
 	"github.com/nyaruka/goflow/utils"
 )
 
+type FlowInfo struct {
+	Dependencies *Dependencies `json:"_dependencies"`
+	Results      []*ResultInfo `json:"_results"`
+	WaitingExits []ExitUUID    `json:"_waiting_exits"`
+}
```

Comment:
Where is this object used exactly? Should the default json keys here not have the leading `_` since that is handled in the flow envelope case?

Reply:
Ah yes.. probably should have written a test for that! This is what `/mr/flow/inspect` would return. We can add whatever we want in here.